src/SFA.DAS.ApplyService.Application/Apply/Roatp/RoatpPreambleQuestionBuilder.cs

```diff
@@ -317,13 +319,30 @@ public static class DeliveringApprenticeshipTraining
             public const string ProfessionalDevelopmentPolicy = "7700";
             public const string ImproveEmployeeSectorExpertise = "7710";
             public const string ImproveEmployeeKnowledge = "7720";
+            public const string DeliveringTrainingOther = "other";
 
             public static class AgricultureEnvironmentalAndAnimalCare
```

Comment:
Can these be moved into their own file(s)? This RoatpPreambleQuestionBuilder file is becoming a lot more than its original purpose